Misc/NEWS.d/next/Documentation/2020-06-20-19-47-29.bpo-19670.wMt__Y.rst

```diff
@@ -0,0 +1 @@
+added documentation to `SimpleCookie.value_decode` and `SimpleCookie.value_encode`
```

Comment:
According to the Dev Guide https://devguide.python.org/contrib/core-team/committing/#changes-that-require-news-entries, docs changes don't requires a news entry file.